native/cocos/platform/interfaces/modules/canvas/ICanvasRenderingContext2D.h

```diff
@@ -31,37 +31,32 @@
 #include "math/Geometry.h"
 #include "platform/interfaces/OSInterface.h"
 
-enum class CanvasTextAlign {
-    LEFT,
-    CENTER,
-    RIGHT
-};
-
-enum class CanvasTextBaseline {
-    TOP,
-    MIDDLE,
-    BOTTOM,
-    ALPHABETIC
-};
 
 namespace cc {
 
-//class CanvasGradient {
-//public:
-//    CanvasGradient();
-//    ~CanvasGradient(); // NOLINT(performance-trivially-destructible)
-//    void addColorStop(float offset, const ccstd::string &color);
-//};
-
 class CC_DLL ICanvasGradient {
 public:
+    ICanvasGradient()                                                 = default;
     ICanvasGradient()                                                   = default;
     virtual ~ICanvasGradient()                                          = default; // NOLINT(performance-trivially-destructible)
     virtual void addColorStop(float offset, const ccstd::string &color) = 0;
 };
 
 class CC_DLL ICanvasRenderingContext2D : public OSInterface {
 public:
+    enum class CanvasTextAlign {
+        LEFT,
+        CENTER,
+        RIGHT
+    };
+
+    enum class CanvasTextBaseline {
```

Comment:
Is `Canvas` prefix needed?

Reply:
Yes, it is more appropriate to remove the canvas head